package.json

```diff
@@ -5,13 +5,24 @@
   "main": "app.js",
   "scripts": {
     "start": "node app.js",
-    "setup": "yarn && cd front-end/ && yarn && yarn build && cd .. && node app.js"
+    "setup": "yarn && cd front-end/ && yarn && yarn build && cd .. && node app.js",
+    "lint": "./node_modules/.bin/eslint '*.js' --fix && ./node_modules/.bin/eslint './front-end/src/**/*.js' --fix"
```

Comment:
It should work to just refer to `eslint` directly, since it's in our dependencies :)

Reply:
Oh, you are right! You learn something new every day :)